schema.json

```
@@ -5,57 +5,54 @@
   "description": "Schema for multiscale pyramid information in Zarr. Properties define the hierarchical layout and transformations.",
   "type": "object",
   "properties": {
-    "version": {
-      "type": "string",
-      "description": "Multiscales metadata version",
-      "pattern": "^0\\.1\\.0$"
-    },
     "layout": {
       "type": "array",
       "description": "Array of objects representing the pyramid layout and transformation relationships",
       "items": {
         "type": "object",
         "properties": {
-          "group": {
+          "asset": {
             "type": "string",
-            "description": "Group name for this resolution level"
+            "description": "Path to the Zarr group or array for this resolution level. Can be a simple name (e.g., '0') for a child group, or a path with '/' separator for nested groups or arrays (e.g., '0/data' for an array within a group)",
+            "pattern": "^(?!/)(?!.*(\\.\\.))([^/]+(/[^/]+)*)$"
           },
-          "from_group": {
+          "derived_from": {
             "type": "string",
-            "description": "Source group used to generate this level"
-          },
-          "factors": {
-            "type": "array",
-            "description": "Array of decimation factors per axis (e.g., `[2, 2]` for 2x decimation)",
-            "items": {
-              "type": "number"
-            }
-          },
-          "scale": {
-            "type": "array",
-            "description": "Array of scale factors per axis describing the resolution change from the derived group",
-            "items": {
-              "type": "number"
-            }
+            "description": "Path to the source Zarr group or array used to generate this level. Uses the same path syntax as 'asset' with '/' separator for nested resources",
+            "pattern": "^(?!/)(?!.*(\\.\\.))([^/]+(/[^/]+)*)$"
           },
-          "translation": {
-            "type": "array",
-            "description": "Array of translation offsets per axis in the coordinate space",
-            "items": {
-              "type": "number"
-            }
+          "transform": {
+            "type": "object",
+            "description": "Transformation parameters describing the coordinate transformation for this level. Can contain scale/translation, proj:transform/proj:shape, or other convention-specific transformation parameters.",
+            "properties": {
+              "scale": {
+                "type": "array",
+                "description": "Array of scale factors per axis describing the resolution change from the derived group",
```

```suggestion
                "description": "Array of scale factors per axis describing the coordinate transformation from the source level (derived_from) to this level",
```
in the case of a translation only